docs/querying/sql.md

```diff
@@ -363,6 +366,45 @@ All 'array' references in the multi-value string function documentation can refe
 | `MV_TO_STRING(arr,str)` | joins all elements of arr by the delimiter specified by str |
 | `STRING_TO_MV(str1,str2)` | splits str1 into an array on the delimiter specified by str2 |
 
+### Sketch operators
+
+These functions operate on expressions or columns that return sketch objects.
+
+#### HLL sketch operators
+
+The following functions operate on [DataSketches HLL sketches](../development/extensions-core/datasketches-hll.html).
+The [DataSketches extension](../development/extensions-core/datasketches-extension.html) must be loaded to use the following functions.
+
+|Function|Notes|
+|--------|-----|
+|`HLL_SKETCH_ESTIMATE(expr)`|Returns the distinct count estimate from an HLL sketch. `expr` must return an HLL sketch.|
+|`HLL_SKETCH_ESTIMATE_WITH_ERROR_BOUNDS(expr, [numStdDev])`|Returns the distinct count estimate and error bounds from an HLL sketch. `expr` must return an HLL sketch. An optional `numStdDev` argument can be provided.|
+|`HLL_SKETCH_UNION([lgK, tgtHllType], expr0, expr1, ...)`|Returns a union of HLL sketches, where each input expression must return an HLL sketch. The `lgK` and `tgtHllType` can be optionally specified as the first parameter; if provided, both optional parameters must be specified.|
+|`HLL_SKETCH_TO_STRING(expr)`|Returns a human-readable string representation of an HLL sketch for debugging. `expr` must return an HLL sketch.|
+
+#### Theta sketch operators
+
+The following functions operate on [theta sketches](../development/extensions-core/datasketches-theta.html).
+The [DataSketches extension](../development/extensions-core/datasketches-extension.html) must be loaded to use the following functions.
+
+|Function|Notes|
+|--------|-----|
+|`THETA_SKETCH_ESTIMATE(expr)`|Returns the distinct count estimate from a theta sketch. `expr` must return a theta sketch.|
+|`THETA_SKETCH_ESTIMATE_WITH_ERROR_BOUNDS(expr, errorBoundsStdDev)`|Returns the distinct count estimate and error bounds from a theta sketch. `expr` must return a theta sketch.|
+|`THETA_SKETCH_UNION([size], expr0, expr1, ...)`|Returns a union of theta sketches, where each input expression must return an theta sketch. The `size` can be optionally specified as the first parameter.|
```

Comment:
Typo: an theta sketch -> a theta sketch

(same for following 2 lines)

FYI, it's possible that the spellcheck in travis will have some false positives on the function names. You can add suppress false positives for this file by adding entries after https://github.com/apache/incubator-druid/blob/master/website/.spelling#L1307

Reply:
thanks, fixed, looks like there were no spellcheck errors